src/Observable.ts

```diff
@@ -96,9 +97,10 @@ export class Observable<T> implements CoreOperators<T>  {
       }
     } else {
       const next = <((x?) => void)> observerOrNext;
-      subscriber = Subscriber.create(next, error, complete);
+      subscriber = Subscriber.create(next, error, complete, start);
     }
 
+    subscriber.start(subscriber);
     subscriber.add(this._subscribe(subscriber));
```

Comment:
In es-observable, I'm not calling the subscribe function if the subscription is already closed.  I think it would be surprising for users to receive a closed subscription.

```js
new Observable(subscriber => {
    // I'll probably write this assuming that subscriber represents an active subscription
});
```